.ci/Jenkinsfile

```diff
@@ -121,6 +121,8 @@ pipeline {
             dir("${BASE_DIR}"){
               withMageEnv(){
                 withCredentials([string(credentialsId: "${GITHUB_TOKEN_CREDENTIALS}", variable: 'GITHUB_TOKEN')]) {
+                  // Ensure that tags are present so goreleaser can build the changelog from the last release.
+                  gitCmd(cmd: 'fetch', args: '--tags')
```

Comment:
This should work, but if for some reason it does not, you can use:

```
steps {
   deleteDir()
   checkout([$class: 'GitSCM', branches: [[name: 'main']], extensions: [[$class: 'CheckoutOption', timeout: 10], [$class: 'CloneOption', noTags: false, reference: '', shallow: false]], userRemoteConfigs: [[credentialsId: 'f6c7695a-671e-4f4f-a331-acdce44ff9ba', url: 'https://github.com/elastic/elastic-package/']]])
   withMageEnv(){
     ...
```

If you go to the [pipeline-syntax](https://beats-ci.elastic.co/pipeline-syntax/) and look for the `checkout step` you can build the step through the UI

<img width="392" alt="image" src="https://user-images.githubusercontent.com/2871786/161801891-7c9b26ed-03a9-48eb-82fe-baf64c23aee6.png">